recipes/mimalloc/all/conanfile.py

```diff
@@ -0,0 +1,149 @@
+        self._cmake.definitions["MI_SECURE"] = self.options.secure
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        if self.settings.compiler == "Visual Studio" and self.settings.arch == "x86":
+            tools.replace_path_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                                       "mimalloc-redirect.lib", "mimalloc-redirect32.lib")
+        with tools.vcvars(self.settings) if self.settings.compiler == "Visual Studio" else tools.no_op():
+            cmake = self._configure_cmake()
+            cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        with tools.vcvars(self.settings) if self.settings.compiler == "Visual Studio" else tools.no_op():
+            cmake = self._configure_cmake()
+            cmake.install()
+
+        if self.settings.os == "Windows" and self.options.shared:
+            if self.settings.arch == "x86_64":
+                self.copy("mimalloc-redirect.dll", src=os.path.join(self._source_subfolder, "bin"),
+                          dst=self._install_prefix)
+            elif self.settings.arch == "x86":
+                self.copy("mimalloc-redirect32.dll", src=os.path.join(self._source_subfolder, "bin"),
+                          dst=self._install_prefix)
+
+        tools.rmdir(os.path.join(self.package_folder, self._install_prefix, "cmake"))
+
+    @property
+    def _install_prefix(self):
+        version = tools.Version(self.version)
+        return os.path.join("lib", "{}-{}.{}".format(self.name, version.major, version.minor))
+
+    @property
+    def _obj_name(self):
+        name = "mimalloc"
+        if self.options.secure:
+            name += "-secure"
+        if self.settings.build_type not in ("Release", "RelWithDebInfo", "MinSizeRel"):
+            name += "-{}".format(str(self.settings.build_type).lower())
+        return name
+
+    @property
+    def _lib_name(self):
+        name = "mimalloc"
+        if self.settings.os == "Windows" and not self.options.shared:
+            name += "-static"
+        if self.options.secure:
+            name += "-secure"
+        if self.settings.build_type not in ("Release", "RelWithDebInfo", "MinSizeRel"):
+            name += "-{}".format(str(self.settings.build_type).lower())
+        return name
+
+    def package_info(self):
+        self.cpp_info.includedirs = [os.path.join(self._install_prefix, "include")]
+        if self.options.get_safe("single_object"):
+            obj_ext = "obj" if self.settings.os == "Windows" else "o"
+            obj_file = "{}.{}".format(self._obj_name, obj_ext)
+            obj_path = os.path.join(self.package_folder, self._install_prefix, obj_file)
+            self.cpp_info.exelinkflags = [obj_path]
```

Comment:
No `sharedlinkflags`? (for shared libraries)

Reply:
Fixed in ec32d8e6